docs/src/content/docs/id/reference/frontmatter.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Referensi Frontmatter
+description: Ikhtisar bidang frontmatter default yang didukung Starlight.
+---
+
+Anda dapat menyesuaikan masing-masing halaman Markdown dan MDX di Starlight dengan menetapkan nilai di frontmatter. Sebagai contoh, halaman biasa dapat mengatur bidang `title` dan `description`:
+
+```md
+---
+title: Tentang proyek ini
+description: Pelajari lebih lanjut tentang proyek yang sedang Saya kerjakan.
+---
+
+Selamat datang di halaman Tentang!
+```
+
+## Bidang Frontmatter
+
+### `title` (diperlukan)
+
+**type:** `string`
```

Comment:
Just to double check, this is fine to have "type" here, it translates well and not something that may have been missed?